flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java

```diff
@@ -81,6 +84,22 @@ public void testRedisSetDataType() throws Exception {
         jedis.del(REDIS_KEY);
     }
 
+    @Test
+    public void testRedisStringDataTypeWithTTL() throws Exception {
```

Comment:
Don't you think we should add the command to the test name?

Reply:
Following this test class pattern, every test is specific for each type. I thought this was awkward, but I don't know if 'mixing' what is being tested is a good option. What do you think?

Reply:
But all their tests have the redis command in the name check the previous one it's testing Set and it has Set on the name